content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,446 @@
+---
+title: "Veneers"
+linkTitle: "Veneers (alpha)"
+weight: 5
+date: 2022-06-30
+---
+
+>Note: `veneers` are **ALPHA** functionality and may adopt breaking changes
+
+
+## Concept and Design
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale. We introduce the concept of a `veneer` as an approach to simplifying interacting with FBCs.
+
+In this context, there are two components to every `veneer`:
+1. An arbitrary API
```

Comment:
It feels a bit unclear what is an arbitrary API in this context. Should we provide an example?

Reply:
Maybe, for instance an input yaml file that respects an arbitrary schema

Reply:
I was deliberately trying to avoid being opinionated here.  The critical aspect is that whatever it is, in #2 it must generate a valid FBC. 
For e.g. all these are valid `veneers` when they output valid FBC:
- XSLT which processes internal cataloging system XML
- build process which knows how to insert a new build
- script to take a text list of operators in explicit order and generate the channel edges for a template

"A veneer is anything that can be converted through an arbitrary process into a valid FBC" didn't have the same gravitas. 

I'm open to anything that doesn't narrow the definition too much, since the critical aspect is the end-state, not the start-state.